src/main/java/com/mraof/minestuck/block/EnumCassetteType.java

```diff
@@ -1,24 +1,57 @@
 package com.mraof.minestuck.block;
 
+import com.mraof.minestuck.util.MSSoundEvents;
+import net.minecraft.sounds.SoundEvent;
+import net.minecraft.sounds.SoundEvents;
 import net.minecraft.util.StringRepresentable;
 
 public enum EnumCassetteType implements StringRepresentable
 {
 	NONE,
+	ELEVEN,
 	THIRTEEN,
-	CAT,
 	BLOCKS,
+	CAT,
 	CHIRP,
+	DANCE_STAB_DANCE,
+	EMISSARY_OF_DANCE,
 	FAR,
 	MALL,
 	MELLOHI,
-	EMISSARY_OF_DANCE,
-	DANCE_STAB_DANCE,
-	RETRO_BATTLE_THEME;
+	PIGSTEP,
+	RETRO_BATTLE_THEME,
+	STAL,
+	STRAD,
+	WAIT,
+	WARD;
 	
 	@Override
 	public String getSerializedName()
 	{
 		return name().toLowerCase();
 	}
+	
+	public static SoundEvent getSoundEvent(EnumCassetteType type)
+	{
+		return switch(type)
```

Comment:
It should be appropriate to define a field on the enum instead of this switch block.

Reply:
Ok, it should be right now